pygments/lexers/make.py

```diff
@@ -193,6 +193,7 @@ class CMakeLexer(RegexLexer):
         ],
         'ws': [
             (r'[ \t]+', Whitespace),
+            (r'(?:#\[(?P<level>=*)\[[\w\W]*?\](?P=level)\])', Comment),
```

Comment:
The outer group `(?:...)` shouldn't be necessary here...

Reply:
Thanks, adjusted